erica/application/tax_declaration/TaxDeclarationService.py

```diff
@@ -0,0 +1,46 @@
+from abc import ABCMeta, abstractmethod
+from uuid import UUID
+
+from opyoid import Module
+from erica.application.EricaRequest.EricaRequestService import EricaRequestService
+from erica.application.Shared.response_dto import JobState
+from erica.application.Shared.response_state_mapper import map_status
+from erica.application.tax_declaration.tax_declaration_dto import EstResponseDto, ResultEstResponseDto
+
+
+class TaxDeclarationServiceInterface:
+    __metaclass__ = ABCMeta
+
+    @abstractmethod
+    def get_response_send_est(self, request_id: UUID) -> EstResponseDto:
+        pass
```

Comment:
If all those Interfaces had one common base class, we could put  `erica_request_service.get_request_by_request_id(request_id)` and error handling for the JobTypeDoesNotMatchEndpointError into that base class and use it for all of the different services. Wdyt?

Reply:
I have just added a base_service, have a look!